lib/order.rb

```diff
@@ -23,7 +23,7 @@ def total
 
   def complete
     calculate_total
-    send_text("Thank you for your order: £#{@total}")
+    send_text("Thank you! Your order was placed and will be delivered before #{(Time.now + (60 * 60)).strftime("%k:%M")}.")
```

Comment:
great that you had a step in between and added the time option once everything worked well together :)